omod-common/src/test/java/org/openmrs/module/webservices/rest/web/filter/ContentTypeFilterTest.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.filter;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletResponse;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.mock.web.MockFilterChain;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Tests for the {@link ContentTypeFilter} class.
+ */
+public class ContentTypeFilterTest {
+	
+	private ContentTypeFilter testFilter;
+	
+	private MockFilterChain mockChain;
+	
+	private MockHttpServletRequest req;
+	
+	private MockHttpServletResponse resp;
+	
+	@Before
+	public void init() {
+		testFilter = new ContentTypeFilter();
+		mockChain = new MockFilterChain();
+		req = new MockHttpServletRequest();
+		resp = new MockHttpServletResponse();
+	}
+	
+	@Test
+	public void doFilter_shouldFilterXMLContent() throws IOException, ServletException {
```

Comment:
Since we have used "Allow" for the other tests, can we consistently use the same even for this test?
When i first looks at "Filter" in the name, it was not obvious to me whether it is to filter to allow or not to allow.